src/drivers/actuators/voxl_esc/qc_esc_packet.c

```diff
@@ -151,6 +153,36 @@ int32_t qc_esc_create_rpm_packet4_fb(int16_t rpm0, int16_t rpm1, int16_t rpm2, i
 	return qc_esc_create_packet(ESC_PACKET_TYPE_RPM_CMD, (uint8_t *) & (data[0]), 10, out, out_size);
 }
 
+int32_t qc_esc_create_rpm_div2_packet4_fb(int32_t rpm0, int32_t rpm1, int32_t rpm2, int32_t rpm3,
+				     uint8_t led0, uint8_t led1, uint8_t led2, uint8_t led3, 
+					 int32_t fb_id, uint8_t *out, uint16_t out_size)
+{
+	int16_t data[5];
```

Comment:
Why do you declare it as int16_t and then below cast as a uint16_t before the assignment?

Reply:
Good catch!! It should be cast to `int16_t`, will fix now. I can remove the cast if you prefer since the data array is already `int16_t`.